platforms/hyperledger-besu/configuration/roles/create/crypto/clique/tasks/check_vault.yaml

```diff
@@ -0,0 +1,15 @@
+# This task checks for the crypto material in the vault
+- name: Check the crypto material in Vault
+  shell: |
+    vault kv get -format=json {{ vault.secret_path | default('secret') }}/{{ component_ns }}/crypto/{{ item.name }}/data
```

Comment:
This command will fail when secretsv2 is used. Please check existing code on how to update them